schema.sql

```diff
@@ -48,3 +48,12 @@ CREATE TABLE visits(
 
 -- Add an email column to your owners table
 ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+-- Create index on animals_id for improved perfomance
+CREATE INDEX visits_animal_id_asc ON visits(animal_id ASC);
+
+-- CREATE index on visits for vet_id
+CREATE INDEX visits_vet_id_asc ON visits(vet_id ASC);
+
+-- CREATE index on owners email
+CREATE INDEX owners_email_asc ON owners(email ASC);
```

Comment:
- [ ] [REQUIRED] Good job so far! The second screenshot should show how much time the query takes `AFTER` the improvements introduced. The `execution` time should be significantly lower than 1 sec (1000 ms) but your `AFTER` result is very high than 1 sec. Please fix this issue and update `screenshot` on your `PR` description 🙏

![image](https://user-images.githubusercontent.com/63915024/200401383-887c0d54-44af-4ef0-9aad-ec2f875b0f5c.png)